perf/CMakeLists.txt

```diff
@@ -30,8 +53,17 @@ create_perf_test(PREFIX tuple
                  SOURCES tuple.cc ${PROJECT_SOURCE_DIR}/test/unit/box_test_utils.c
                  LIBRARIES core box tuple benchmark::benchmark
 )
+create_perf_test_target(PREFIX tuple)
 
 create_perf_test(PREFIX light
                  SOURCES light.cc ${PROJECT_SOURCE_DIR}/test/unit/box_test_utils.c
                  LIBRARIES small benchmark::benchmark
 )
+create_perf_test_target(PREFIX light)
+
+create_perf_test_target(PREFIX small)
+
+add_custom_target(test-c-perf
+                  DEPENDS ${RUN_PERF_C_TEST}
```

Comment:
Minor: I forgot to mention it for the previous commit, but this is valid for both (i.e. Lua and C) patches: I'd rather use `RUN_PERF_<LANG>_TESTS_LIST` to emphasize the CMake variable type. Or use plural for `*_TESTS` at least...

Reply:
renamed in both patches